lib/membrane_live/room.ex

```diff
@@ -303,44 +306,24 @@ defmodule MembraneLive.Event do
     {:noreply, state}
   end
 
-  def handle_info({:timer_action, action}, %{timer: timer} = state) do
-    case Timer.handle_action(timer, action,
-           timeout: MembraneLive.get_env!(:last_peer_timeout_long_ms)
-         ) do
-      {:ok, timer} ->
-        {:noreply, %{state | timer: timer}}
+  @impl true
+  def handle_info(:close_room, state) do
+    result = Engine.terminate(state.rtc_engine, timeout: 10_000, force?: true)
 
-      {:timeout, _timer} ->
-        close_webinar(state)
+    if result == {:error, :timeout} do
+      Membrane.Logger.warn(
+        "[Event: #{state.event_id}] RTC Engine was forced kill. This can cause some problems with playing HLS playlist."
```

Comment:
Do we know precisely what the HLS playlist issues are?

Reply:
Let's say that we use ll-hls if we force kill hls endpoint recorded playlist will have partials at the end which is not good and potentially can cause some problem as it is vod (not live) playlist.